examples/src/examples/graphics/lines.tsx

```diff
@@ -19,6 +54,29 @@ class LinesExample {
         assetListLoader.load(() => {
             app.start();
 
+            // Create a shader definition and shader from the vertex and fragment shaders
+            // for the dashed lines material
+            const shaderDefinition = {
+                attributes: {
+                    aPosition: pc.SEMANTIC_POSITION,
+                    aTAndLength: pc.SEMANTIC_TEXCOORD0,
+                    aColor: pc.SEMANTIC_COLOR
+                },
+                vshader: files['shader.vert'],
+                fshader: files['shader.frag']
+            };
+            const shader = new pc.Shader(app.graphicsDevice, shaderDefinition);
+
+            // Create a new material with the new shader
+            const material = new pc.Material();
+            material.shader = shader;
+            material.blend = true;
```

Comment:
I believe `.blend` is deprecated. You want to use `Material.blendState` instead.

Reply:
@LeXXik that's from an old commit, its changes are reverted. If you look at `Filed changed` tab for this PR you'll notice that there are no changes to the `lines.tsx` file, I believe it doesn't even exist anymore being replaced by a `lines.mjs` file in the examples.